python/ingestion/dataset_utils.py

```diff
@@ -1,61 +1,147 @@
 import pandas as pd  # type: ignore
+import numpy as np  # type: ignore
 from ingestion import gcs_to_bq_util
 import ingestion.standardized_columns as std_col
 import ingestion.constants as constants
 
+from ingestion.standardized_columns import Race
 
-def generate_pct_share_col(df, raw_count_col, pct_share_col, breakdown_col, all_val):
-    """Returns a DataFrame with a percent share row based on the raw_count_col
+
+def generate_pct_share_col_without_unknowns(df, raw_count_to_pct_share, breakdown_col, all_val):
+    """Returns a DataFrame with a percent share row based on the raw_count_cols
```

Comment:
`with a percent share row based on the` -> "with a percent share `column` based on the"  right?